app/views/docs/graphql.phtml

```diff
@@ -0,0 +1,529 @@
+            The data returned by the query, maps requested field names to their results.
+        </td>
+    </tr>
+    <tr>
+        <td data-title="Name: ">
+            errors
+        </td>
+        <td data-title="Type: ">
+            object[]
+        </td>
+        <td data-title="Description: ">
+            An array of errors that occurred during the request.
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+<p>The data object will contain a map of requested field names to their results. If no data is returned, the object will not be present in the response.</p>
+
+<p>The errors array will contain error objects, each with their own <b>message</b> and <b>path</b>. The path will contain the field key that is null due to the error. If no errors occur, the array will not be present in the response.</p>
+
+<h2><a href="/docs/realtime#authentication" id="authentication">Authentication</a></h2>
+<p>GraphQL authenticates using Appwrite accounts and sessions. Both accounts and sessions can be created with GraphQL using the <code>accountCreate</code>, <code>accountCreateEmailSession</code>, <code>accountCreateAnonymousSession</code>, or <code>accountCreatePhoneSession</code> mutations.</p>
+
+<p>More information and examples of authenticating users can be found in the dedicated <a href="/docs/authentication">authentication guide</a>.</p>
+
+<h2><a href="/docs/graphql#graphql-vs-rest" id="graphql-vs-rest">GraphQL vs REST</a></h2>
+
+<p>There are two main features that make GraphQL appealing when compared to the REST API: <b>selection sets</b> and <b>query batching.</b></p>
+
+<h3><a href="/docs/graphql#selection-sets" id="selection-sets">Selection Sets</a></h3>
+
+<p>Selection sets can be used to tell a GraphQL API exactly which fields of a particular resource you would like to receive in the response. The server will respond with only those fields, nothing more, nothing less. This gives you full control over what data comes into your application.</p>
+
+<p>For example, to retrieve only the email of a currently authenticated user, you could query the <code>accountGet</code> field, passing only email as the <b>field selection set</b>.</p>
+
+<div class="ide margin-bottom" data-lang="graphql" data-lang-label="GraphQL">
+    <pre class="line-numbers"><code class="prism language-graphql" data-prism>query GetAccount {
+    accountGet {
+        email
+    }
+}</code></pre></div>
+
+<p>Given this query, the GraphQL API will respond with:</p>
+
+<div class="ide margin-bottom" data-lang="json" data-lang-label="JSON">
+    <pre class="line-numbers"><code class="prism language-json" data-prism>{
+    "data": {
+        "accountGet": {
+            "email": "..."
+        }
+    }
+}</code></pre></div>
+
+<p>This can be an extremely useful feature for performance, network efficiency, and app responsiveness. As the processing happens on the server, the bandwidth consumed for the request can be dramatically reduced.</p>
```

Comment:
```suggestion
<p>This can be a useful feature for performance, network efficiency, and app responsiveness. As the processing happens on the server, the bandwidth consumed for the request can be dramatically reduced.</p>
```

Let's avoid hyperbolic adjectives in documentation, makes us sound less professional and trustworthy.